code/modules/client/preference_setup/vore/01_ears.dm

```diff
@@ -100,7 +113,10 @@
 		if(selection && selection != "Normal")
 			pref.ear_style = pretty_ear_styles[selection]
 		else
-			pref.ear_style = null
+			if(pref.ear_style)
+				return TOPIC_REFRESH_UPDATE_PREVIEW
```

Comment:
I have an alternate suggestion for this part.
`var/list/pretty_ear_styles = list("Normal")` should be `var/list/pretty_ear_styles = list("Normal" = null)` instead, so that if they pick normal, it just sets it to null, and you can drop the entire 'else' statement entirely, as well as the `selection != "Normal"`